data/language/nl-NL.yml

```
@@ -1849,13 +1849,13 @@ strings:
   1811: Bouwt {STRINGID} in de buurt van {STRINGID}
   1812: Bouwt {STRINGID} in de buurt van {STRINGID}
   1813: Bouwt {STRINGID} in de buurt van {STRINGID}
-  1814: Checking services near {STRINGID}
-  1815: Surveying landscape near {STRINGID}
+  1814: Onderzoekt de voorzieningen in de buurt van {STRINGID}
+  1815: Onderzoekt het landschap in de buurt van {STRINGID}
```

'Bezichtigt de omgeving van {STRINGID}'?

---

Ik vind 'Onderzoeken' een wat fijnere (vrijere) vertaling, maar ik zou alsnog een kortere string willen voorstellen. Een beetje een compromis van beide:
```suggestion
  1815: Onderzoekt de omgeving van {STRINGID}
```